packages/plugin-modtools/config/robo.mjs

```diff
@@ -13,8 +15,11 @@ export default {
 			'GuildBans',
 			'GuildMessages',
 			'GuildModeration',
-			'MessageContent'
-		]
+			'MessageContent',
+			'GuildMembers'
+		],
+		partials: [Partials.Channel, Partials.Message, Partials.GuildMember]
```

Comment:
Is partials necessary for this? I'm hesitant to include these in the plugin by default as that will cause users using it to start receiving events and data they otherwise wouldn't have received. I'd rather users opt-in instead.

Reply:
I'm pretty sure the messages partical is need as it used the message create event. But I think I can remove the other particals. I just added all the ones my test bot had.

Reply:
Would you mind testing without those partials to verify please?